app/routines/newflow/create_or_update_salesforce_lead.rb

```diff
@@ -61,9 +61,9 @@ def exec(user:)
 
 
       if user.salesforce_lead_id
-        lead = OpenStax::Salesforce::Remote::Lead.find_by(email: user.best_email_address_for_salesforce)
+        lead = OpenStax::Salesforce::Remote::Lead.find_by(email: "#{user.best_email_address_for_salesforce}")
```

Comment:
@Dantemss do you think this will help with [this issue](https://openstax.sentry.io/issues/5414102669/?environment=production&project=5772471&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D&referrer=issue-stream&sort=date&statsPeriod=24h&stream_index=3)? Would `to_s` achieve the same?
Seems to be exploding the the email address has a `'` in it.

Reply:
`best_email_address_for_salesforce` already returns a string or nil, so I'm not sure that would help here.

Reply:
reverted this, sounds like we need to look into fixes elsewhere
[for reference, slack message](https://openstax.slack.com/archives/C0R8GMF1S/p1729528884970799)